csharp/PublicAPI/net471/PublicAPI.Shipped.txt

```diff
@@ -831,6 +831,13 @@ static ParquetSharp.Arrow.ArrowReaderProperties.GetDefault() -> ParquetSharp.Arr
 static ParquetSharp.Arrow.ArrowWriterProperties.GetDefault() -> ParquetSharp.Arrow.ArrowWriterProperties!
 static ParquetSharp.Column.CreateSchemaNode(ParquetSharp.Column![]! columns, ParquetSharp.LogicalTypeFactory! logicalTypeFactory, string! nodeName = "schema") -> ParquetSharp.Schema.GroupNode!
 static ParquetSharp.Column.CreateSchemaNode(ParquetSharp.Column![]! columns, string! nodeName = "schema") -> ParquetSharp.Schema.GroupNode!
+static ParquetSharp.DateTimeNanos.FromDotnetTicks(long dotnetTicks) -> ParquetSharp.DateTimeNanos
```

Comment:
I think it's fine to leave these in `*.Shipped.txt` now, but the idea is they should go into the `*.Unshipped.txt` files and only be moved to `Shipped.txt` when a stable release is made.

Reply:
Ah right - sorry! As you can see I haven't used this before ... will remember for next time!